screens/auth/LoginScreen.js

```diff
@@ -107,25 +135,62 @@ export default class Login extends React.Component {
       .catch(err => console.error(err));
   }
 
+  onFocus(signUpField) {
+    const { indicators } = this.state;
+    indicators[signUpField] = fieldStateColors.FOCUSED;
+    this.setState({
+      indicators
+    });
+  }
+
+  onBlur(signUpField) {
+    const { indicators } = this.state;
+    indicators[signUpField] = fieldStateColors.BLURRED;
+    this.setState({
+      indicators
+    });
+  }
+
   render() {
     return (
-      <LoginContainer>
-        <Input
-          placeholder="Phone Number (i.e. 1234567890)"
-          keyboardType="number-pad"
-          maxLength={10}
-          value={this.state.phoneNumber}
-          onChangeText={text => this.setState({ phoneNumber: text })}
-        />
-        <Input
-          placeholder="Password"
-          secureTextEntry
-          onChangeText={text => this.setState({ password: text })}
-          value={this.state.password}
-        />
-        <Button title="Log In" onPress={() => this.handleSubmit()} />
+      <AuthScreenContainer>
+        <BigTitle>Log in</BigTitle>
+        <FormContainer>
+          <AuthTextField
+            fieldType="Phone Number"
+            color={this.state.indicators[signUpFields.PHONENUM]}
+            value={this.state.phoneNumber}
+            onBlurCallback={() => this.onBlur(signUpFields.PHONENUM)}
+            onFocusCallback={() => this.onFocus(signUpFields.PHONENUM)}
+            changeTextCallback={text => this.setState({ phoneNumber: text })}
+          />
+          <AuthTextField
+            fieldType="Password"
+            color={this.state.indicators[signUpFields.PASSWORD]}
+            value={this.state.password}
+            onBlurCallback={() => this.onBlur(signUpFields.PASSWORD)}
+            onFocusCallback={() => this.onFocus(signUpFields.PASSWORD)}
+            changeTextCallback={text => this.setState({ password: text })}
+          />
+        </FormContainer>
+        <JustifyCenterContainer>
+          <FilledButtonContainer
+            style={{ marginTop: 104 }}
+            width="100%"
+            onPress={() => this.handleSubmit()}>
+            <ButtonLabel color="#fff">LOG IN</ButtonLabel>
```

Comment:
Same nit as below: this should be normally cased due to the `text-transform` in `ButtonLabel`'s styling